### ISSUE TITLE
Warning popup when using "Tabs with Dropdown" generated `id` and `aria-controls` attributes for the sake of proper component accessibility

### ISSUE THREAD

**Issue by mrmons** [react-bootstrap/react-bootstrap#2338]
I follow the sample provider by here 

https://react-bootstrap.github.io/components.html#tabs
I am using the "Tab with Dropdown"

And I get this error
browser.js:49
Warning: In the context of a `<TabContainer>`, `<NavItem>`s are given generated `id` and `aria-controls` attributes for the sake of proper component accessibility. Any provided ones will be ignored. To control these attributes directly, provide a `generateChildId` prop to the parent `<TabContainer>`.

Issue been file before but it seen like been fixed? But maybe not "Tab with Dropdown"?
https://github.com/react-bootstrap/react-bootstrap/issues/1854

**Comment by priand** [comment]
I'm seeing something similar with 0.31.2:
Warning: Failed prop type: The prop `id` is required to make `Tabs` accessible for users of assistive technologies such as screen readers.
